integration-test/scripts/check-results.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 #please don't add this property to Jenkins
-: ${INTEGRATIONTEST_MAX_PG_NETWORK_OUTPUT:="4.05GB"}
+: ${INTEGRATIONTEST_MAX_PG_NETWORK_OUTPUT:="4.17GB"}
```

Comment:
this is for using flow id during polling https://jira.cloudera.com/browse/CB-17922 when this ticket is implemented, output limit could be decreased again